### ISSUE TITLE
[OTHER] Twitter Widget - Fails WCAG 2.1

### ISSUE THREAD

**Issue by argallel** [wet-boew/wet-boew#9604]
There are multiple WCAG 2.1 fails in the twitter widget, as listed below:
1. Focus order from social media links go to the i link before the twitterhandle link which is out of order. 
2. The announcement on the i link info and privacy is announced twice on a screen reader.
3. When a user clicks the link a new window is opened without notifying the user visually or programmatically.
4. The 'decorative' image is announcing alternative text. This is not necessary and should be alt="".
5. Like, and Share tweet linked icons are too small and neither are keyboard accessible.
6. Embed link is confusing a user will not know what this is.
7. View on twitter is redundant as various links are are ready indicated for this page. 

These issues are present on Wetboew 4.0.56.6 and are seen in chrome and firefox on a regular screen size.
